### PR TITLE
Docs: Need connect null values content

### PR DESCRIPTION
**Pull request by oddlittlebird**
@dprokop, I need content and images for these options.

### CODE REVIEW

**Comment by achatterjee-grafana** [comment]
Is this ready for review?

**Comment by oddlittlebird** [comment]
> Is this ready for review?

Not until @dprokop adds content.

**Comment by pkolyvas** [comment]
Added some details. Here's the accompanying image:

connect-null-values-8-0

**Comment by oddlittlebird** [comment]
All the places that say "Description" need to have the description of the field entered.

**Comment by pkolyvas** [comment]
@oddlittlebird Thanks - sorry I missed those. Chalk it up to my first-timer-ism. 

I took a stab at them and fixed an inaccuracy from my previous edit.

File: docs/sources/panels/visualizations/time-series/graph-time-series-as-lines.md [hunk 1]
```diff
@@ -146,21 +146,23 @@ Dot spacing set to 0, 30:
...
 
-If there is a gap in the series, the line in the graph will be broken and show the gap.
+#### Never
 
-![Null values gaps example](/static/img/docs/time-series-panel/line-graph-null-gaps-7-4.png)
+Time series data points with gaps in the the data are never connected.
+#### Always
 
-#### Connected
+Time series data points with gaps in the the data are always connected.
```
Comment (oddlittlebird) [inline docs/sources/panels/visualizations/time-series/graph-time-series-as-lines.md:160]:
```suggestion
Time series data points with gaps in the the data are always connected.

```

File: docs/sources/panels/visualizations/time-series/graph-time-series-as-lines.md [hunk 2]
```diff
@@ -146,21 +146,23 @@ Dot spacing set to 0, 30:
...
+Choose how null values (gaps in the data) are displayed on the graph. Null values can be connected to form a continuous line or, optionally, set a threshold above which gaps in the data should no longer be connected. 
 
-#### Gaps
+![Image name](/static/img/docs/time-series-panel/connect-null-values-8-0.png)
 
-If there is a gap in the series, the line in the graph will be broken and show the gap.
+#### Never
 
-![Null values gaps example](/static/img/docs/time-series-panel/line-graph-null-gaps-7-4.png)
+Time series data points with gaps in the the data are never connected.
```
Comment (oddlittlebird) [inline docs/sources/panels/visualizations/time-series/graph-time-series-as-lines.md:157]:
```suggestion
Time series data points with gaps in the the data are never connected.

```

**Review by oddlittlebird — COMMENTED** [review]
A couple suggestions for adding space

**Comment by oddlittlebird** [comment]
@pkolyvas, this looks good. Could you approve it?

**Review by pkolyvas — APPROVED** [review]
LGTM!

**Pull request merged**